docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md

```diff
@@ -0,0 +1,411 @@
+
+```shell
+oc apply \  
+  -n yb-platform \  
+  -f - <<EOF
+apiVersion: yugabyte.com/v1alpha1
+kind: YBPlatform
+metadata: 
+  name: ybplatform-sample
+spec: 
+  image:  
+    repository: registry.connect.redhat.com/yugabytedb/yugabyte-platform  
+    tag: latest 
+  ocpCompatibility:  
+    enabled: true 
+  rbac:  
+    create: false
+EOF
+
+# output
+ybplatform.yugabyte.com/ybplatform-sample created
+```
+
+To verify that the pods of the Yugabyte Platform instance are in Running state, execute the following:
+
+```shell
+oc get pods -n yb-platform -l app=ybplatform-sample-yugaware
+
+# output
+NAME                         READY  STATUS  RESTARTS  AGE
+Ybplatform-sample-yugaware-0  5/5   Running  0        22s
+```
+
+### Upgrading the Yugabyte Platform Instance
+
+You may choose to upgrade the Yugabyte Platform instance installed using the Operator to a new tag that you receive from Yugabyte. In the current release, you can do this by using the command line. 
+
+The following example shows the command you would execute to update the container image tag to 2.5.2.0-b89:
+
+```shell
+oc patch \
+ ybplatform ybplatform-sample \
+ -p '{"spec":{"image":{"tag":"2.5.2.0-b89"}}}' --type merge \
+ -n yb-platform
+ 
+# output
+ybplatform.yugabyte.com/ybplatform-sample patched
+```
+
+To verify that the pods are being updated, execute the following command:
+
+```shell
+oc get pods -n yb-platform -l app=ybplatform-sample-yugaware -w
+
+# output
+NAME                         READY  STATUS          RESTARTS  AGE
+ybplatform-sample-yugaware-0  5/5   Running            0     18m
+ybplatform-sample-yugaware-0  0/5   Terminating        0     19m
+ybplatform-sample-yugaware-0  0/5   Pending            0     0s
+ybplatform-sample-yugaware-0  0/5   ContainerCreating  0     35s
+ybplatform-sample-yugaware-0  5/5   Running            0     93s
+```
+
+## Helm-Based Installation
+
+In addition to meeting the requirements described in  [Prepare the OpenShift Environment](../../../install-yugabyte-platform/prepare-environment/openshift/), you need to perform the following steps before attempting to install Yugbyte Platform using Helm:
+
+- Verify that the OpenShift cluster is configured with Helm 3 or later by executing the following command:
+
+  ```shell
+  $ helm version
+  ```
+
+  The output should be similar to the following:
+
+  ```
+  version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"
+  ```
+
+- Ensure that an OpenShift secret license file has been obtained from Yugabyte Support.
+
+Installing Yugabyte Platform on an OpenShift cluster using Helm involves the following:
+
+- [Creating an instance of Yugabyte Platform](#creating-an-instance-of-yugabyte-platform-via-helm)
+- [Finding the availability zone labels](#finding-the-availability-zone-labels)
+- [Configuring the CLI with the OCP cluster](#configuring-the-cli with-the-ocp-cluster)
```

Comment:
```suggestion
- [Configuring the CLI with the OCP cluster](#configuring-the-cli-with-the-ocp-cluster)
```

Reply:
I fixed this already. It'll be visible when the PR is commited

Reply:
This is still the same even with your latest pushed commit. Can you please check once.

Reply:
There were three of these links, and one was broken. Thanks for catching it, @bhavin192 !